R/drive_publish.R

```diff
@@ -78,8 +78,7 @@ drive_change_publish <- function(file = NULL, publish = TRUE, ..., verbose = TRU
     proc_res <- process_response(response)
   }
 
-  ## TO DO: do we really have to call the API again?
-  file_update <- as_dribble(drive_id(file$id))
+  file_update <- drive_id(file$id)
```

Comment:
It still feels like there is at least one gratuitous API call in the typical publishing workflow.

1. Get current metadata.
1. Main API call to enact new publishing status.
1. Optional API call re: auto-publishing.
1. Get current metadata again? This is the one that feels unnecessary to me.

Can you take a look?